src/mono/mono/mini/mini-wasm-debugger.c

```diff
@@ -64,11 +65,13 @@ extern void mono_wasm_add_properties_var (const char*, gint32);
 extern void mono_wasm_add_array_item (int);
 extern void mono_wasm_set_is_async_method (guint64);
 extern void mono_wasm_add_typed_value (const char *type, const char *str_value, double value);
+extern void mono_wasm_add_files(const char *assembly_data, guint32 assembly_len, const char *pdb_data, guint32 pdb_len);
```

Comment:
nit: space before '('


```suggestion
extern void mono_wasm_add_files (const char *assembly_data, guint32 assembly_len, const char *pdb_data, guint32 pdb_len);
```

Reply:
Thanks! I've resolved your comments because I've addressed the issues on my local branch. Will push once I've gotten more feedback from others.